extra/3-magic-8-ball.js

```diff
@@ -43,10 +43,54 @@
     Very doubtful.
 */
 
+
+const veryPositive = [
+  "It is certain.",
+  "It is decidedly so.",
+  "Without a doubt.",
+  "Yes - definitely.",
+  "You may rely on it.",
+];
+
+const positive = [
+  "As I see it, yes.",
+  "Most likely.",
+  "Outlook good.",
+  "Yes.",
+  "Signs point to yes.",
+];
+
+const negative = [
+  "Reply hazy, try again.",
+  "Ask again later.",
+  "Better not tell you now.",
+  "Cannot predict now.",
+  "Concentrate and ask again.",
+];
+
+const veryNegative = [
+  "Don't count on it.",
+  "My reply is no.",
+  "My sources say no.",
+  "Outlook not so good.",
+  "Very doubtful.",
+];
+
+const answers = [
+  ...veryPositive,
+  ...positive,
+  ...negative,
+  ...veryNegative,
+];
+
 // This should log "The ball has shaken!"
 // and return the answer.
 function shakeBall() {
   //Write your code in here
+  console.log("The ball has shaken!");
+  let randomAnswer = answers[Math.floor(Math.random() * answers.length)];
+  
+  return randomAnswer;
```

Comment:
This is clearly written, nice one. When I run it locally though I have a failing test. Have just seen you haven't had a chance to finish this yet. Am interested to see where you go with it if you get a chance :).

Reply:
It's strange, because everything is working properly